### DISCUSSION TITLE
Scheduler performance tests?

### DISCUSSION THREAD

**Discussion by rosegayming** [Frogging-Family/linux-tkg#630] [Q&A]
Has anyone benchmarked different schedulers? Struggling to find them for TaskType and BORE. 

Interested for gaming, though realise it may be pointless and smarter to optimize in other areas first...

**Comment by AlexanderWKoenig** [comment]
Hi, made yesterday some benchmarks in latest Cyberpunk 2077 benchmark mode with 4 different tkg kernels:
https://flightlessmango.com/logs/4521

**Reply by ptr1337** [reply]
> Hi, made yesterday some benchmarks in latest Cyberpunk 2077 benchmark mode with 4 different tkg kernels: https://flightlessmango.com/logs/4521

Seems equal to my results. Just my TT Scheduler Benchmarks were not that well. Around 3 % over PDS.
